presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestTransformCorrelatedScalarAggregationToJoin.java

```diff
@@ -147,14 +139,11 @@ public void rewritesOnSubqueryWithProjection()
                                                 values(ImmutableMap.of("a", 0, "b", 1)))))));
     }
 
-    private AggregationNode createSumAggregation(PlanBuilder p, Symbol symbol, List<List<Symbol>> groupingSets, PlanNode source)
+    private AggregationNode createSumAggregation(PlanBuilder p, List<List<Symbol>> groupingSets, PlanNode source)
```

Comment:
I wonder if the method invocation is not more difficult to understand than then method body. I would inline this method then, thanks to that there would be no need to pass empty List<List> arguments as there will be no need to call `groupingSets` method